tests/director/tests/application_templates_subscription_test.go

```diff
@@ -0,0 +1,110 @@
+package tests
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	"github.com/kyma-incubator/compass/tests/pkg/certs/certprovider"
+	"github.com/kyma-incubator/compass/tests/pkg/fixtures"
+	"github.com/kyma-incubator/compass/tests/pkg/gql"
+	"github.com/kyma-incubator/compass/tests/pkg/subscription"
+	"github.com/kyma-incubator/compass/tests/pkg/tenant"
+	"github.com/kyma-incubator/compass/tests/pkg/testctx"
+	testingx "github.com/kyma-incubator/compass/tests/pkg/testing"
+	"github.com/kyma-incubator/compass/tests/pkg/token"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSubscriptionApplicationTemplateFlow(stdT *testing.T) {
```

Comment:
That test only the "subscribe" scenario. Enhance a little bit the test at the end to make "unsubscribe" and to validate there is no application of the application template in the given tenant